docs/illink-tasks.md

```diff
@@ -1,5 +1,13 @@
 # ILLink.Tasks
 
+## Terminology
+
+The capability of the SDK is called "trim". Trimming allows the SDK to reduce the size of the application by removing unused code and other assets. MSBuild/SDK integration should use "trim" in all user facing names (property names, item group names, ...).
```

Comment:
Nit: this sentence is a bit confusing because it reads like "the SDK" is the .NET SDK, and singular "capability" implies that the SDK has only one capability, namely trimming. Could you instead say: "The ILLink tasks provide the ability to "trim" a set of published assemblies"?.